mmdt/mmdt/settings.py

```diff
@@ -177,27 +177,24 @@
 LOGGING = {
     'version': 1,
     'disable_existing_loggers': False,
-    'handlers': {
-        'timed_rotate_file': {
-            'class': 'logging.handlers.TimedRotatingFileHandler',
-            'filename': os.path.join(log_dir, 'django_timed.log'),
-            'when': 'midnight',  # Rotate log file at midnight, effectively once a day
-            'interval': 1,  # Interval set to 1, combined with 'midnight' means once a day
-            'backupCount': 30,  # Keep last 5 files
-            'formatter': 'verbose',
-        },
-    },
     'formatters': {
         'verbose': {
-            'format': '{levelname} {asctime} {module} {process:d} {thread:d} {message}',
+            'format': '{levelname} {asctime} {module} {message}',
             'style': '{',
         },
     },
-    'loggers': {
-        'django': {
-            'handlers': ['timed_rotate_file'],
-            'level': 'INFO',
-            'propagate': True,
+    'handlers': {
+        'rotate_file': {
+            'level': 'DEBUG',
+            'class': 'logging.handlers.RotatingFileHandler',
+            'filename': os.path.join(log_dir, 'django.log'),
+            'maxBytes': 10 * 1024 * 1024,
+            'backupCount': 5,
+            'formatter': 'verbose',
```

Comment:
I think that are not your change, please be sure you create the branch from main branch. We have already update those in main branch